src/types/auditLog.ts

```diff
@@ -0,0 +1,17 @@
+export enum AuditableActivity {
```

Comment:
aye shall we not use enums hahah

https://dev.to/ivanzm123/dont-use-enums-in-typescript-they-are-very-dangerous-57bh#:~:text=Excessive%20use%20of%20regular%20ENUMs,new%20values%20can%20be%20added.

Reply:
Adjusted in 06dbe93cdd89d76d2ed8be5280d966bbdf2a47db.